### ISSUE TITLE
portrait status  - png not working

### ISSUE THREAD

**Issue by gleckler1** [PCMDI/pcmdi_metrics#355]
@doutriaux1 
First test with portrait using conda.... pdf looks good but png is blank.
Is there already a way can add a title to the top of the portrait or is that for v1.2?
[test.pdf](https://github.com/PCMDI/pcmdi_metrics/files/285184/test.pdf)

**Comment by doutriaux1** [comment]
hum... Travis is using the conda and the pngs work. Although you're on mac, so it might be different. Thanks,

**Comment by gleckler1** [comment]
@doutriaux1 produced it on oceanonly ... is the command still
x.png('crap')
x.pdf('crap')

**Comment by doutriaux1** [comment]
using conda?

**Comment by gleckler1** [comment]
@doutriaux1  we ... I' mean oui

**Comment by gleckler1** [comment]
@doutriaux1 ... x.png works with x=vcs.init(bg=1,geometry=(2400,3200)), but with x = vcs.init() it does not.

**Comment by durack1** [comment]
@gleckler1 got a UV-CDAT version number for us?
```
import cdat_info
print cdat_info.version()
```

**Comment by gleckler1** [comment]
print cdat_info.version()
['', '2.4.1']

**Comment by durack1** [comment]
@gleckler1 I too have hit issues with `2.4.1` the current `master` branch has a bunch of fixes underway that are getting closer.. Can you build a `nightly` version of UV-CDAT locally to be testing against?

```
conda create –n uvcdatNightly –c uvcdat/label/release –c uvcdat uvcdat
```
That should at least get you back up and running and generating png files

**Comment by doutriaux1** [comment]
@gleckler1 @durack1 is this still true?

**Comment by durack1** [comment]
@gleckler1 can answer that..

**Comment by gleckler1** [comment]
@doutriaux1  I will check after 7:30-9am telecon, and its ok if its not fixed for 1.1.1.   Before trying to deal with it please first address 
https://github.com/PCMDI/pcmdi_metrics/pull/373
and
https://github.com/PCMDI/pcmdi_metrics/issues/367
I need these fixed before I can finalize demos for last 1.1.1 pull request
